### ISSUE TITLE
User story field

### ISSUE THREAD

**Issue by lukefretwell** [ScanGov/data#13]
@aaronhans per our convo about 'anecdotes' ... user stories!

Format:

`As a [type of user], I want [some goal] so that [some reason].`

Examples:

- Contrast: `As a visually impaired user, I want sufficient color contrast between text and background so that I can read the content without strain or missing information.`
- security.txt: `As a security researcher, I want to find a security.txt file on the website so that I can report vulnerabilities to the right contact securely and efficiently.`
- robots.txt: `As a search engine crawler, I want to read the robots.txt file so that I know which parts of the website I am allowed or disallowed to index.`

**Comment by aaronhans** [comment]
Largest Contentful Paint: As a commuter on the BART train riding through Lafayette, I want your website to display the main content efficiently so my temporarily constrained bandwidth which radically exacerbates loading times doesn't make me give up before it loads.

Interaction To Next Paint: As a user of a cheaper model mobile phone, I want your website to respond to my interactions quickly. If it is inefficient and performs a lot of processing on my device I wonder if it is even working, get stressed out and may click multiple times.

**Comment by lukefretwell** [comment]
@aaronhans:

- LCP: https://standards.scangov.org/lcp/
- INP: https://standards.scangov.org/inp/

**Comment by lukefretwell** [comment]
These are in the standards website. We can continue to iterate, so feel free to update how you see fit.
